WORKSPACE

```diff
@@ -37,6 +37,11 @@ local_repository(
     path = "internal/e2e/packages",
 )
 
+local_repository(
```

Comment:
This is not being used yet. `/packages` could get added to the root `.bazelignore` and we don't need this local repository yet.